site/src/docs/components/header/index.mdx

```diff
@@ -106,5 +116,21 @@ Coming soon!
         <Header.NavigationLink href="#" label="Link 3" />
       </Header.NavigationMenu>
     </Header>
+<<<<<<< HEAD
```

Comment:
Erh... some rebasing leftovers here -> remove 😄

Reply:
fixed now and rebased to `hds-1803-header-dark-theme-support`